server/bundler/utils.js

```diff
@@ -14,6 +14,9 @@ function getAssets( stats ) {
 }
 
 function pathToRegExp( path ) {
+	if ( path === '/' ) {
+		return path;
+	}
 	return new RegExp( '^' + path + '(/.*)?$' );
```

Comment:
> we could drop the RegEx here

that sounds like a great idea to me. even better, and related to the 404 stuff, would be for us to reject paths like `/design/*` and instead require that we have a whitelist of URLs

```js
// something like this
[ '/design', '/design/:thing', '/design/things/:id' ]
```